index.js

```diff
@@ -10,11 +10,10 @@ const md5 = require("md5");
 const isString = require("lodash.isstring");
 const cloneDeep = require("lodash.clonedeep");
 const version = require("./package.json").version;
-
 const setImmediate = global.setImmediate || process.nextTick.bind(process);
 const noop = () => {};
 
-class Analytics {
+export default class Analytics {
```

Comment:
You can't use require and export in the same file, can you?

Reply:
Not sure about this. The original file had `module.exports = Analytics` which was giving me TypeScript errors. I'll replace it in the file instead of export here.